doc/release/1.9.0-notes.rst

```diff
@@ -26,42 +26,117 @@ For running on PyPy, PyPy3 6.0+ is required.
 Highlights of this release
 **************************
 
+- Added `scipy.optimize.milp`, new function for mixed-integer linear
+  programming.
+- Added `scipy.stats.fit` for fitting discrete and continuous distributions
+  to data.
+- Added new spline based interpolation methods for
+  `scipy.interpolate.RegularGridInterpolator`.
+- A new global optimizer (DIviding RECTangles algorithm)
+  `scipy.optimize.direct`
+
 
 ************
 New features
 ************
 
+Lazy access to subpackages
+==========================
+
+Before this release, all subpackages of SciPy (`cluster`, `fft`, `ndimage`,
+etc.) had to be explicitly imported. Now, these subpackages are lazily loaded
+as soon as they are accessed, so that the following is possible:
+`import scipy as sp; sp.fft.dct([1, 2, 3])`. Advantages include: making it
+easier to navigate SciPy in interactive terminals, reducing subpackage import
+conflicts (which before required
+`import networkx.linalg as nla; import scipy.linalg as sla`),
+and avoiding repeatedly having to update imports during teaching &
+experimentation. Also see
+[the related community specification document](https://scientific-python.org/specs/spec-0001/).
+
 `scipy.cluster` improvements
 ============================
 
 
 `scipy.interpolate` improvements
 ================================
-
+- Speed up the ``RBFInterpolator`` evaluation with high dimensional
+interpolants.
+- Added new spline based interpolation methods for
+`scipy.interpolate.RegularGridInterpolator` and its tutorial.
+- `scipy.interpolate.RegularGridInterpolator` and `scipy.interpolate.interpn`
+now accept descending ordered points.
+- ``RegularGridInterpolator`` now handles length-1 grid axes
```

Comment:
Now that gh-16220 is in, let's add

- `BivariateSpline`s have grown a new method `partial_derivative` which constructs a new spline object representing a derivative of an original spline. This mirrors the corresponding functionality for univariate splines, `splder` and `BSpline.derivative`, and can substantially speed up repeated evaluation of derivatives.